spec/factories/users.rb

```diff
@@ -1,9 +1,14 @@
 FactoryBot.define do
   factory :user do
+    email { Faker::Internet.email }
     first_name { Faker::Name.first_name }
     last_name  { Faker::Name.last_name }
-    email { Faker::Internet.email }
+    avatar { Faker::File.file_name }
```

Comment:
no you would not get avatar that way. You need to create folder in spec directory where you store some random avatars, then you assign them here

Reply:
teammate who implements User crud said he would do it)